src/DropdownMenu.js

```diff
@@ -97,16 +97,16 @@ class DropdownMenu extends React.Component {
       ...props,
     } = this.props;
 
-    delete props.bsRole;
+    const [bsProps, elementProps] = splitBsPropsAndOmit(props, ['bsRole']);
```

Comment:
We could... I don't know. I didn't add it there because nothing else in `bootstrapUtils` dealt with `bsRole` at all. Do you think it makes sense there?